4.12 content update + variables

dialvare opened the pull request.

dialvare committed:
Update welcome.md

dialvare committed:
Update using-homeroom.md

dialvare committed:
Update welcome.md

dialvare committed:
Update welcome.md

dialvare committed:
Add files via upload

dialvare committed:
Add files via upload

dialvare committed:
Update explore-openshift.md

dialvare committed:
Update welcome.md

dialvare committed:
Update validation.md

dialvare committed:
Add files via upload

dialvare committed:
Add files via upload

dialvare committed:
Add files via upload

dialvare committed:
Add files via upload

dialvare committed:
Add files via upload

dialvare committed:
Update deploy-cnv.md

dialvare committed:
Update deploy-cnv.md

dialvare committed:
Update deploy-cnv.md

dialvare committed:
Update deploy-cnv.md

dialvare committed:
Update storage-setup.md

dialvare committed:
Update network-setup.md

dialvare committed:
Update deploy-workloads.md

dialvare committed:
Update live-migration.md

dialvare committed:
Update cloning.md

dialvare committed:
Update explore-openshift.md

dialvare committed:
Update explore-openshift.md

dialvare committed:
Update validation.md

dialvare committed:
Delete ocp-virt-hco-4-new.png

dialvare committed:
Add files via upload

dialvare committed:
Add files via upload

dialvare committed:
Update deploy-cnv.md

dialvare committed:
Update storage-setup.md

dialvare committed:
Update deploy-workloads.md

dialvare committed:
Add files via upload

dialvare committed:
Add files via upload

dialvare committed:
Update live-migration.md

dialvare committed:
Update cloning.md

dialvare committed:
Update templates.md

dialvare committed:
Add files via upload

dialvare committed:
Delete storage-project.png.png

dialvare committed:
Add files via upload

dialvare committed:
Add files via upload

dialvare committed:
Update templates.md

dialvare committed:
Add files via upload

dialvare committed:
Add files via upload

dialvare committed:
Add files via upload

dialvare committed:
Add files via upload

dialvare committed:
Add files via upload

dialvare committed:
Update deploy-application-components.md

dialvare committed:
Add files via upload

dialvare committed:
Add files via upload

dialvare committed:
Update deploy-database-web.md

dialvare committed:
Add files via upload

dialvare committed:
Update deploy-database-yaml.md

dialvare committed:
Add files via upload

dialvare committed:
Add files via upload

dialvare committed:
Add files via upload

dialvare committed:
Add files via upload

dialvare committed:
Add files via upload

dialvare committed:
Add files via upload

dialvare committed:
Update snapshot-restore.md

dialvare committed:
Add files via upload

dialvare committed:
Add files via upload

dialvare committed:
Add files via upload

dialvare committed:
Add files via upload

dialvare committed:
Add files via upload

dialvare committed:
Delete hot-plug-disk-lsblk-new.png

dialvare committed:
Add files via upload

dialvare committed:
Add files via upload

dialvare committed:
Add files via upload

dialvare committed:
Update hot-plug.md

dialvare committed:
Update micro-segmentation.md

dialvare committed:
Update snapshot-restore.md

dialvare committed:
Update templates.md

dialvare committed:
Update templates.md

dialvare committed:
Update deploy-cnv.md

dialvare committed:
Update cloning.md

dialvare committed:
Update welcome.md

dialvare committed:
Update deploy-cnv.md

dialvare committed:
Update cloning.md

dialvare committed:
Update storage-setup.md

dialvare committed:
Update network-setup.md

dialvare committed:
Update deploy-workloads.md

dialvare committed:
Update live-migration.md

dialvare committed:
Update templates.md

dialvare committed:
Update deploy-application-components.md

dialvare committed:
Update deploy-database-web.md

dialvare committed:
Update snapshot-restore.md

dialvare committed:
Add files via upload

dialvare committed:
Update hot-plug.md

dialvare committed:
Update cloning.md

dialvare committed:
Add files via upload

dialvare committed:
Update masquerade.md

hgeaydem commented on files/lab/workshop/content/cloning.md:
````diff
@@ -349,12 +349,6 @@ Then enable nginx service:
 systemctl daemon-reload && systemctl enable --now nginx
 ```
 
-Which should show:
````
why removing this ? bit concerned if it doesn't show, means nginx might not be correctly enabled ?

dialvare replied:
Deleted by mistake. Added again in the last commit.

hgeaydem commented on files/lab/workshop/content/cloning.md:
```diff
@@ -581,7 +575,7 @@ NAME         AGE   PHASE     IP               NODENAME                       REA
 ~~~
 
-This machine will also be visible from the OpenShift Virtualization console, which you can navigate to using the top "**Console**" button, or by using your dedicated tab if you've created one. You can login using the "**root**" user with the password you've set earliyer, by going into the "**Workloads**" --> "**Virtualization**" --> "**fc34-clone**" --> "**Console**", if you want to try:
+This machine will also be visible from the OpenShift Virtualization tab. Switch to the **full OpenShift console** (%cnvlab-console-url%). There, you can login using the "**root**" user with the password you've set earlier, by going into the "**Virtualization**" → "**Virtual Machines**" → "**fc34-clone**" → "**Console**", if you want to try:
```
seems to be missing a step there ... they'll have to login to the console with kubeadmin / %cnvlab-kubeadmin-password% and then login with the root user in the VM Console

dialvare replied:
Not sure what you're referring to. I re-wrote the paragraph with the login steps

hgeaydem commented on files/lab/workshop/content/explore-openshift.md:
````diff
@@ -16,21 +16,26 @@ Projects are a top level concept to help you organise your resources. An OpenShi
 ```
 
-When you see the prompt, if required, agree to the SSH certificates by typing "yes", and then enter **%bastion-password%** as the password. Then you can execute following command to get the kubeadmin password:
+When you see the prompt, if required, agree to the SSH certificates by typing "yes", and then enter the bastion password: %cnvlab-bastion-password%. Then you can execute the following command to get the kubeadmin password:
````
this might not be needed anymore since we can reference the %cnvlab-kubeadmin-password% var directly now and this is not really a standard way to recover a lost openshift kubeadmin passwd either ;)

dialvare replied:
Agree. Steps deleted in the new commit :)

dialvare committed:
Deleting steps getting kubeadmin from bastion 

Deleted steps to get the kubeadmin pass from the bastion machine

dialvare committed:
Add files via upload

dialvare committed:
Add files via upload

dialvare committed:
Add files via upload

dialvare committed:
Adding symlink nginx confirmation 

Added Created symlink /etc/systemd/system/multi-user.target.wants/nginx.service → /etc/systemd/system/nginx.service. confirmation and rewrote Vm console login steps.

hgeaydem reviewed (approved):
LGTM
